rocketc/rocketc.py

```diff
@@ -290,7 +309,29 @@ def _update_user(self, user_id, username, email):
         if email != self.email:
             url_path = "users.update"
             data = {"userId": user_id, "data": {"email": email}}
-            response = self.request_rocket_chat("post", url_path, data)
+            response = self._request_rocket_chat("post", url_path, data)
             if response["success"]:
                 self.email = email
         self._set_avatar(username)
+
+    @XBlock.json_handler
+    def set_default_channel(self, data, suffix=""):
+        """
+        This method set the default variable for the channels
+        """
+        # pylint: disable=unused-argument
+        default_channel = data["channel"]
+        if default_channel != " " and default_channel != None:
```

Comment:
Here the second part of the conditional should be "and default_channel is not None"

Reply:
ok I'll fix it